Library/Format.swift

```diff
@@ -403,7 +403,7 @@ public enum Format {
 
 public let defaultThresholdInDays = 30 // days
 
-internal struct DateFormatterConfig {
+internal struct DateFormatterConfig: Hashable, Equatable {
```

Comment:
Cool, yea let's do this - according to the documentation - `Hashable` conformance requires `Equatable`, so you probably only need `Hashable`.

Reply:
Cool, I'll delete the explicit Equatable. Thanks!